update etcd runbooks (#5)
kubesphere/alert-runbooks: content/runbooks/etcd/etcdGRPCRequestsSlow.md

#### Console dashboards

> In the KubeSphere console "cluster status" page under "Monitoring & Alering", select the etcd dashboard. There are both RPC rate as well as Disk Sync Duration dashboards

Reviewer: ```suggestion
In KubeSphere console's "cluster status" page under `Monitoring & Alering`, select the `etcd` dashboard. There are both RPC rate as well as Disk Sync Duration dashboards
```

To confirm this is the cause of the slow requests either:

> 1. In KubeSphere console "cluster status" page under "Monitoring & Alering", select the overview dashboard and view the

Reviewer: ```suggestion
1. In KubeSphere console's `cluster status` page under `Monitoring & Alering`, select the `overview` dashboard and take a look at the
```

   CPU Usage.

> 2. In KubeSphere console cluster "overview", view "Resource Usage Ranking" to see top consumers of CPU.

Reviewer: ```suggestion
2. In KubeSphere console's `cluster overview`, in the `Resource Usage Ranking` panel you can take a look at the top consumers of CPUs.
```
